components/sections/Navbar/index.tsx

```diff
@@ -0,0 +1,76 @@
+import * as AccessibleIcon from '@radix-ui/react-accessible-icon';
+import Image from 'next/image';
+import { useTheme } from 'next-themes';
+import { useMemo } from 'react';
+
+import LanguageDropdown from '@/components/Common/LanguageDropDown';
+import ThemeToggle from '@/components/Common/ThemeToggle';
+import NavItem from '@/components/sections/NavItem';
+import { useLocale } from '@/hooks/useLocale';
+import { useNavigation } from '@/hooks/useNavigation';
+
+import style from './index.module.css';
+
+const Navbar: React.FC = () => {
```

Comment:
```suggestion
const NavBar: FC = () => {
```

^ import the `FC` type, don't rely on React globals.